File: example/src/main/java/de/friedger/remoteconfig/example/remoteconfig/RemoteApplication.java
package de.friedger.remoteconfig.example.remoteconfig;

import android.app.Application;

import de.friedger.remoteconfig.RemoteConfig;

public class RemoteApplication extends Application {
    private static final int REMOTE_CONFIG_VERSION = 1;

    @Override
    public void onCreate() {
        super.onCreate();
        RemoteConfig.getInstance().init(getApplicationContext(), REMOTE_CONFIG_VERSION, true);
    }
}

File: example/src/main/java/is/gangverk/example/remoteconfig/RemoteApplication.kt
package `is`.gangverk.example.remoteconfig

import android.app.Application

import de.friedger.remoteconfig.RemoteConfig

class RemoteApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        RemoteConfig.getInstance()!!.init(applicationContext, REMOTE_CONFIG_VERSION, true)
    }

    companion object {
        private val REMOTE_CONFIG_VERSION = 1
    }
}
